PYME/Acquire/frameWrangler.py

```diff
@@ -54,6 +54,31 @@
 #sfrom PYME.ui import mytimer
 
 class FrameWrangler(object):
+    """
+    Grabs frames from the camera buffers
+
+    Notes
+    -----
+    dispatch Signals are used to allow other files to listen to key events 
+    happinging within the acquisition.
+
+    Attributes
+    ----------
+    onFrame : dispatch.Signal
+        Called once per new-frame appearing in our buffer; used to pass
+        frame data to e.g. spoolers. Note that while onFrame gets called once
+        per new frame, the new frames are only checked for once per polling
+        cycle, meaning that this event will be fired N-times for each 
+        `onFrameGroup` event.
+    onFrameGroup : dispatch.Signal
+        Called on each new frame group (once per polling interval) - use for 
+        updateing GUIs etc.
+    onStop : dispatch.Signal
+    _polling_interval : float
```

Comment:
It's a bit odd having a private attribute with no way of setting documented (leading underscore implies that it should not be accessed from outside the class)

Reply:
This is also a bit of a 'here be dragons' type thing - I wouldn't want to encourage people to change this if it were to be made a public attribute as the results might not be super predictable.